### ISSUE TITLE
Banned for a reason I don't quite understand.

### ISSUE THREAD

**Issue by gotammo** [CreeperHost/CreeperHostGui#252]
I recently was banned from the chat for Inappropriate Conduct, yet I don't recall saying anything that might have qualified as such. I was banned from what I believe to have been Bob, but I am unsure. I don't recall any of my messages containing content that is inappropriate, though since I was banned I obviously had. My last message was yesterday, and likely the most inappropriate that I can easily recall is light swearing (examples being dafuq, hell, and fuck) yet I do not believe I said those yesterday. 

In other words, I do not believe I was banned for a logical reason, or if it was, I cannot deduce why. I am requesting an unban, and whatever I did wrong I will not do again.

Ban ID: mtg-ban=5ebcfdae16e734-53664359

Thank you for your consideration, I have loved MineTogether for a long time and have made many friends whilst online, and I would hate to not be able to meet them again.

**Comment by CH-Paul-T** [comment]
References to a meme that references male genitalia at 4:22am BST 14/05/2020.

You were banned by @Draylynn, as such I'll let her make any decisions.

**Comment by Draylynn** [comment]
Considering your chat history and Discord participation. We'll have the ban removed.
Be careful what you say in chat.

**Comment by gotammo** [comment]
At the time of me posting, I was talking in general, not just about that "meme". I hadn't realized how much of an issue it was of me posting that.

I will keep that in mind.
